sopel/plugins/handlers.py

```diff
@@ -75,33 +76,35 @@ class AbstractPluginHandler(object):
     (commands, jobs, etc.), configuring it, and running any required actions
     on shutdown (either upon exiting Sopel or unloading that plugin).
     """
+
+    @abc.abstractmethod
     def load(self):
         """Load the plugin.
 
         This method must be called first, in order to setup, register, shutdown,
         or configure the plugin later.
         """
-        raise NotImplementedError
 
+    @abc.abstractmethod
     def reload(self):
         """Reload the plugin.
 
         This method can be called once the plugin is already loaded. It will
         take care of reloading the plugin from its source.
         """
-        raise NotImplementedError
 
-    def get_label(self):
+    @abc.abstractmethod
+    def get_label(self) -> str:
```

Comment:
Seems like adding type hints should be mentioned in the commit message, at least.

Reply:
Did that. I had to rebase to do so.